ggshield/cmd/auth/login.py

```diff
@@ -99,7 +145,17 @@ def login_cmd(
     check_instance_has_enabled_flow(config=config)
```

Comment:
I tested in local when passing --instance, the instance is set in the config later so `check_instance_has_enabled_flow` won't check for the correct instance.

I think `check_instance_has_enabled_flow` should not take config as arg but instance instead.

Note: I am not sure that you can easily test sso but you can check the normal flow with staging instance by passing `--instance https://dashboard.staging.gitguardian.com`

Reply:
ok, @martinfaucheux I'll move this in the `web` if block